pkg/storage/storage.go

```diff
@@ -193,8 +193,6 @@ func (s *Storage) LockRelease(name string) error {
 // If release doesn't exist or wasn't previously locked - the unlock will pass
 func (s *Storage) UnlockRelease(name string) {
 	s.Log("unlocking release %s", name)
-	s.releaseLocksLock.Lock()
```

Comment:
@technosophos It's possible that I don't understand something. The only place in which releaseLocks map is being written to is line 186 (the one you marked), which is protected by lock acquired at line 163. Can you show me in what situation can concurrent writes to map occur?